components/popover/popover-mixin.js

```diff
@@ -70,14 +58,15 @@ export const PopoverMixin = superclass => class extends superclass {
 				border-radius: var(--d2l-popover-border-radius, var(--d2l-popover-default-border-radius));
 				box-shadow: var(--d2l-popover-shadow, var(--d2l-popover-default-shadow));
 				box-sizing: border-box;
+				outline: none;
```

Comment:
This is to currently align with dropdown styles. It is arguably an existing a11y issue with existing dropdowns. I will circle back with Jeff about this, and a separate change may be in order.